profile/allProfiles.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Log in page</title>
+    <link rel="stylesheet" href="../css/custom.css">
```

Comment:
I'd prefer it if you consolidated all your stylesheets into one that you can attach to any html file. You can do this using SCSS to bundle all your CSS fragments together. Just makes things a bit cleaner and more maintainable.

Reply:
done, just liked this better when coding, since I could easier find the location of the css codes when looking in developer tool, instead of jsut seeing main.css or something